src/components/display/Table.tsx

```diff
@@ -292,29 +286,19 @@ const DefaultRowFactory: React.VFC<TRowProps> = ({
 		>
 			<td>
 				<Row mainAlignment={'center'}>
-					{showCheckbox && (selected || (multiSelect && selectionMode)) ? (
-						<Checkbox
+					{showCheckbox && (
+						<StyledCheckbox
 							ref={ckbRef}
 							size={'small'}
 							value={selected}
 							onClick={_onChange}
 							iconColor={(multiSelect && selectionMode) || selected ? 'primary' : 'text'}
+							show={selected || (selectionMode && multiSelect)}
```

Comment:
If we extract the results of `selected || (selectionMode && multiSelect)` in a constant, we could use it also on line 299, increasing, imo, the readability of both the checks